### ISSUE TITLE
Docker Webserver Support

### ISSUE THREAD

**Issue by paulgrahek** [bedrock-viz/bedrock-viz#7]
Based on Initial Project Task [here](https://github.com/bedrock-viz/bedrock-viz/projects/1#card-30217667)
A simple approach would be to create a NGINX webserver with the docker command
```
docker run --name bedrock-viz -p 8080:80 -v /path/to/output-dir:/usr/share/nginx/html:ro -d nginx
```
However, this can cause issues because by default an output directory is not created for each output. This can result in the user accidentally mounting an entire Documents folder with files that are sensitive and now exposed on an insecure port. Creating a subfolder for the output by default seems preferred in this circumstance. Thoughts?

**Comment by jasper-wan** [comment]
using volume may be better, we can create a new volume and then copy all generated files into it, see <https://github.com/moby/moby/issues/25245>

**Comment by paulgrahek** [comment]
Noted. Just remembered file permissions can have issues on Windows & Mac with Docker so it's better to just keep it in a volume. I will create a PR to add a section to the README on how to do that.

**Comment by jasper-wan** [comment]
Merged #8
